data/helpers.d/psql

```diff
@@ -0,0 +1,150 @@
+# Create a master password and set up global settings
+# Please always call this script in install and restore scripts
+#
+# usage: ynh_psql_test_if_first_run
+ynh_psql_test_if_first_run() {
+	if [ -f /etc/yunohost/psql ];
+	then
+		echo "PostgreSQL is already installed, no need to create master password"
+	else
+		pgsql=$(ynh_string_random)
+		pg_hba=""
+		echo "$pgsql" >> /etc/yunohost/psql
```

Comment:
Why >> instead of > ?

Reply:
> can be used too according to the code.

Reply:
?

Using >> instead of > isn't without consequences ! That's why I asked if there's a reason or if it's just a mistake.

Reply:
As the file is used to store the unique password (analogous to mysql handling), it should be a `>`.